cmd/sops/decrypt.go

```diff
@@ -59,6 +67,9 @@ func extract(tree *sops.Tree, path []interface{}, outputStore sops.Store) (outpu
 	if newBranch, ok := v.(sops.TreeBranch); ok {
 		tree.Branches[0] = newBranch
 		decrypted, err := outputStore.EmitPlainFile(tree.Branches)
+		if errors.Is(err, json.BinaryStoreEmitPlainError) {
+			err = fmt.Errorf("%s. %s", err.Error(), notBinaryHint)
```

Comment:
```suggestion
			err = fmt.Errorf("%s\n\n%s", err.Error(), notBinaryHint)
```

maybe?

In addition, I just noticed that in combination with the `common.NewExitError` below, this becomes `Error dumping file: Error emitting binary store: ...`. Which makes me wonder if we should prefer a Go idiomatic error as `BinaryStoreEmitPlainError` (resulting in `Error dumping file: error ...`), even if this would diverge from other errors in the code base.

But this is just nitpicking at this point really.

Reply:
Yes, sounds good. Adjusted in 49a4145d34bea2cd9cfb3fdfa1518c6102eb1094. The CLI output now looks like:
```
Error dumping file: error emitting binary store: no binary data found in tree

This is likely not an encrypted binary file? If not, use --output-type to select the correct output type.
```